### ISSUE TITLE
Dovecot : access error to /var/run/dovecot/stats-writer

### ISSUE THREAD

**Issue by grubshka** [YunoHost/issues#1904]
### Describe the bug

Dovecot errors in mail.err

### Context

- Hardware: *VPS bought online*
- YunoHost version: 4.2.8.3
- I have access to my server: *Through SSH | through the webadmin*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*

### Steps to reproduce

Quite standard installation.
Read the logs in /var/log/mail.log
Found access errors to `/var/run/dovecot/stats-writer` :
```
Oct 10 20:14:50 cloud1 postfix/pipe[2321]: 117F380620: to=<XXXX@cloud1.yyy.fr>, orig_to=<XXXX@yyy.fr>, relay=dovecot, delay=0.2, delays=0.09/0.02/0/0.1, dsn=2.0.0, status=sent (delivered via dovecot service (lda(XXXX@cloud1.yyy.fr,)Error: net_connect_unix(/var/run/dovecot/stats-writer) failed: Permissi)) 
```

### Expected behavior

No error in log files :-)

### Solution

I had to add these lines to /etc/dovecot.conf to fix the problem : 

```
service stats {
    unix_listener stats-reader {
        user = vmail
        group = mail
        mode = 0660
    }

    unix_listener stats-writer {
        user = vmail
        group = mail
        mode = 0660
    }
}
```

**Comment by alexAubin** [comment]
Thanks for the detailed issue :+1: 

I guess it indeed doesn't hurt to add that conf snippet so I yolocommited in https://github.com/YunoHost/yunohost/commit/dab3dc6f370e262ed427a3459aebde35f4f92da6 , to be released in next 4.3.x iteration

Cheers !

**Comment by Pyrrhicv** [comment]
Hours of work!  One simple fix, Dovecot/Postfix/MySql/Fedora 34 - copied into dovecot.conf - hell was over.  Many thanks to grubshka - for sure needs to be added!
